biz.aQute.resolve/src/biz/aQute/resolve/Bndrun.java

```diff
@@ -185,4 +194,100 @@ public Collection<Container> getRunbundles() throws Exception {
 		return super.getRunbundles();
 	}
 
+	private Collection<Container> cache() throws Exception {
+		File ours = getPropertiesFile();
```

Comment:
What about included files in the bndrun file and in the workspace?

In the gradle plugin all of the files factor in to out-of-date checks:

https://github.com/bndtools/bnd/blob/16607b11580986dadf96d162c7cf8555dcbfd9a3/gradle-plugins/biz.aQute.bnd.gradle/src/main/java/aQute/bnd/gradle/BndPlugin.java#L981-L986

Reply:
makes sense